packages/aws/data_stream/inspector/elasticsearch/ingest_pipeline/default.yml

```
@@ -92,6 +234,17 @@ processors:
       field: vulnerability.id
       copy_from: aws.inspector.package_vulnerability_details.vulnerability_id
       ignore_failure: true
+  - set:
+      field: vulnerability.cve
+      tag: set_vulnerability_cve
+      copy_from: aws.inspector.package_vulnerability_details.vulnerability_id
+      ignore_empty_value: true
+      if: ctx.aws?.inspector?.type == 'PACKAGE_VULNERABILITY' && ctx.aws?.inspector?.package_vulnerability_details?.vulnerability_id?.startsWith('CVE') == true
```

```suggestion
      if: ctx.aws?.inspector?.type == 'PACKAGE_VULNERABILITY' && ctx.aws.inspector.package_vulnerability_details?.vulnerability_id?.startsWith('CVE') == true
```
`ctx.aws.inspector` must be non-null by the time we get to the second part of the expression.